src/odemis/driver/orsay.py

```diff
@@ -1280,33 +1178,21 @@ def _updateTemperatureRegulation(self, parameter=None, attributeName="Actual"):
         Reads the state of temperature regulation of the GIS reservoir from the Orsay server and saves it in the
         temperatureRegulation VA
         """
-        if parameter not in (self._gis.RegulationOn, self._gis.RegulationRushOn, None):
+        if parameter not in (self._gis.RegulationOn, None):
             raise ValueError("Incorrect parameter passed to _updateTemperatureRegulation. Parameter should be "
-                             "datamodel.HybridGIS.RegulationOn, datamodel.HybridGIS.RegulationRushOn, or None. "
+                             "datamodel.HybridGIS.RegulationOn, or None. "
                              "Parameter passed is %s." % parameter.Name)
         if not attributeName == "Actual":
             return
 
```

Comment:
Please leave a comment explaining that RegulationRushOn is also available, but it actually doesn't work (yet), and we don't need it.